dev/release/00-prepare-test.rb

```diff
@@ -279,13 +272,6 @@ def test_update_version_post_tag
                         "+Version: #{@release_version}.9000"],
                      ],
                    },
-                   {
-                     path: "r/src/Makevars.win",
-                     hunks: [
-                       ["-VERSION = #{@release_version}",
-                        "+VERSION = #{@release_version}.9000"],
-                     ],
-                   },
```

Comment:
What are these changes about?

Reply:
The `00-prepare.sh` does not modify `Makevars.win` anymore, the generation of this file is now deferred at installation (@nealrichardson please confirm) via the added `configure.win`.

Reply:
This patch makes Makevars.win be generated dynamically, so there's no longer a file that needs to be updated in the release scripts.